src/modules/modal.js

```diff
@@ -0,0 +1,144 @@
+import capitalize, { newDate } from './helpers.js';
+import addComment from './comments.js';
+
+export default async function populate(data, element) {
```

Comment:
- [ ] I would suggest you use the `ES6 arrow function` instead of the `function `keyword. Because this is today's trend, also we can see this rule in project learning objectives. 
![Screenshot (176)](https://user-images.githubusercontent.com/67087664/152325361-27f6ecb1-fb54-4381-a927-cc0114aa00b7.png)